test/javascripts/controllers/topic-test.js.es6

```diff
@@ -281,49 +281,32 @@ QUnit.test("Can split/merge topic", function(assert) {
   const controller = this.subject({ model });
   const selectedPostIds = controller.get("selectedPostIds");
 
-  assert.not(
```

Comment:
I see you're removing a lot of JS tests even though we're adding functionality. I think the new functionality should be tested.

Reply:
JavaScript tests added.